tests.md

```diff
@@ -1,10 +1,10 @@
-# The Events Calendar tests
+# Events Calendar PRO tests
 
 This is a brief and quick guide that's covering the bare essentials needed to set up the tests on your local plugin copy.
 Please refer to [Codeception](http://codeception.com/docs) and [WP Browser](https://github.com/lucatume/wp-browser) documentation for any issue that's not TEC related.
 
 ## Set up
-After cloning the TEX repository on your local machine change directory to the plugin root folder and pull in any needed dependency using [Composer](https://getcomposer.org/):
+After clojning the TEX repository on your local machine change directory to the plugin root folder and pull in any needed dependency using [Composer](https://getcomposer.org/):
```

Comment:
It's a Tibetan tradition of monks trying to misspell any word they come across in a gesture of refusal of modern society tendency toward unachievable perfection in body and mind.